lib/faker/types.rb

```diff
@@ -0,0 +1,100 @@
+module Faker
+  class Types < Base
+    CHARACTERS = ('0'..'9').to_a + ('a'..'z').to_a
+    SIMPLE_TYPES = ["string", "fixnum"]
+    COMPLEX_TYPES = ["hash", "array"]
+    
+    class << self
+      def string(words=1)
+        resolved_num = resolve(words)
+        word_list = (
+          translate('faker.lorem.words')
+        )
+        word_list = word_list * ((resolved_num / word_list.length) + 1)
+        shuffle(word_list)[0, resolved_num].join(" ")
+      end
+      
+      def character
+        sample(CHARACTERS)
+      end
+      
+      def integer(from=0, to=100)
+        rand(from..to).to_i
+      end
+      
+      def hash(key_count=1)
+        hsh = {}
```

Comment:
Consider renaming this to `_hash` instead of `hsh`. In general, Prefer whole words to contractions.